app/validators/csv/facilities_validator.rb

```diff
@@ -29,7 +29,12 @@ def at_least_one_facility
   end
 
   def duplicate_rows
-    fields = facilities.map { |facility| facility.slice(:organization_name, :facility_group_name, :name) }
+    fields = facilities
+      .map { |facility| facility.slice(:organization_name, :facility_group_name, :name) }
+      .map do |facility|
+        facility[:name] = facility[:name].gsub(/\s+/, "")
+        facility
+      end
```

Comment:
Now that we've got more logic in here, we could simplify this a little bit perhaps, using a single map.

```ruby
facilies.map { |facility|
  organization_name: facilities[:organization_name],
  facility_group_name: facilities[:facility_group_name],
  name: facilities[:name].gsub(/\s+/, "")
}
```